help/destinations/catalog/advertising/liveramp-onboarding.md

```diff
@@ -74,6 +77,8 @@ To authenticate to the destination, fill in the required fields and select **[!U
 
 ![Sample screenshot showing how to authenticate to the destination using SSH key](../../assets/catalog/advertising/liveramp-onboarding/liveramp-sftp-ssh.png)
 
+* **[!UICONTROL Port]**: The port used for your [!DNL LiveRamp - Onboarding] storage location.  This is contingent on your geographic region.
```

Comment:
"contingent" is one of those terms that should not be present in clear technical documentation. Please rephrase this, maybe like this:

The port used for your [!DNL LiveRamp - Onboarding] storage location.  Use the port that corresponds to your geographical location, as described below:

- `EU`: `4222`


What about the other regions? What about AU and US? Shouldn't all the regions be mentioned in both cases?

Reply:
Edited to your language.  EU will be the only relevant region here as SSH key authorization is specific to that region (not supported in others)